src/assets/style.css

```diff
@@ -321,6 +321,11 @@ section {
   box-sizing: border-box;
 }
 
+.viewInfo {
```

Comment:
This is used on a <section> to turn off the background color for the new info box